CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Update to component auditing. ([PR 1996](https://github.com/alphagov/govuk_publishing_components/pull/1996))
```

Comment:
Just so everyone can tell what type of update this is:

```suggestion
* Update to component auditing. ([PR 1996](https://github.com/alphagov/govuk_publishing_components/pull/1996)) PATCH
```

Reply:
Thanks @injms !